### DISCUSSION TITLE
Enigmatic Legacy now throws an error as of REI version 8.3.495 (forge)

### DISCUSSION THREAD

**Discussion by dog38145** [shedaniel/RoughlyEnoughItems#960] [Q & A] [answered]
Hello! I seem to have found an incompatibility regarding Enigmatic Legacy and the newest REI update. I am using the latest version of forge and all of my mods according to curse.

https://gist.github.com/dog38145/e2a9c9e0fa6899a09e2863d267f72a69 is my crash log, let me know if there's anything else I can do to help :)

**Comment by dog38145** [comment]
Oh hold on, simply downgrading REI Plugin Compatibilities to 8.2.492 allows the game to launch and for me to join my server.

**Reply by dog38145** [reply]
Not the fix though as tag trees are a jumbled mess of text lol

**Comment by shedaniel** [comment, marked as answer]
I am releasing a new update that fixes this.
